src/user/proposals/AdminProposalDashboard/DashboardContent/DashboardContent.js

```diff
@@ -2,12 +2,143 @@ import React, { Component } from "react";
 import { Button, Form, Col, Image } from "react-bootstrap";
 import "./DashboardContent.scss";
 import userIcon2 from "../../../../images/userIcon2.jpg";
+import { Link } from "react-router-dom";
 
 class DashboardContent extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      token:
```

Comment:
@AuraOfDivinity Please remove this token from here. Already token is saved in the localStorage. Fetch from there after login